src/modules/user/repository/userRepositories.ts

```diff
@@ -0,0 +1,9 @@
+import Users from "../../../databases/models/users"
+import authRepositories from "../../auth/repository/authRepositories"
+
+const UpdateUserByAttributes = async (userId: number, status: string) =>{
```

Comment:
UpdateUserByAttributes does not work in this way. Reach out to @AimePazzo and ask help